app/jobs/users/follow_job.rb

```diff
@@ -0,0 +1,11 @@
+module Users
+  class FollowJob < ApplicationJob
+    queue_as :users_follow
+
+    def perform(user, followable)
```

Comment:
I was conflicted between passing ``user_id``, ``followable_id`` and ``followable_type`` to the active job (which seems to be what's most used in the codebase) and passing the objects ``user`` and ``followable`` directly. I opted for the latter, because it made things simpler/cleaner, but let me know if you think the former is more appropriate.